package.json

```diff
@@ -114,8 +114,8 @@
   "devDependencies": {
     "@hapi/joi": "^17.1.1",
     "@sinonjs/fake-timers": "^6.0.1",
-    "@types/pino": "^6.0.1",
     "@types/node": "^14.10.3",
+    "@types/pino": "^6.0.1",
```

Comment:
I would restore it

Reply:
Yeah, leave `package.json` out of this PR unless new dependencies need to be introduced/removed.

Reply:
Makes sense. Reverted it.